indy_node/test/rs_encoding/test_send_rs_get_encoding.py

```diff
@@ -0,0 +1,159 @@
+    _, identifier = sdk_wallet_trustee
+    authors_did, name, version, type = identifier, "indy_encoding_sha", "1.3", "9"
+    _id = identifier + ':' + type + ':' + name + ':' + version
+    encoding = "UTF-8_SHA-256-2"
+    request_json = build_rs_encoding_request(identifier, encoding, name, version)
+    encoding_json, reply = sdk_write_request(looper, sdk_pool_handle, sdk_wallet_trustee, request_json)
+    return encoding_json, reply
+
+
+def test_send_get_rs_encoding_succeeds(
+        looper, sdk_pool_handle, nodeSet, sdk_wallet_trustee, send_rs_encoding):
+    _, did = sdk_wallet_trustee
+    txnId = send_rs_encoding
+    request = build_get_rs_encoding_request(did, txnId)
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_trustee, request)])
+    assert rep[0][1]['result']['seqNo']
+
+
+def test_send_get_rs_encoding_as_client(
+        looper, sdk_pool_handle, nodeSet, sdk_wallet_client, send_rs_encoding):
+    _, did = sdk_wallet_client
+    txnId = send_rs_encoding
+    request = build_get_rs_encoding_request(did, txnId)
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_client, request)])
+    assert rep[0][1]['result']['seqNo']
+
+
+def test_send_get_rs_encoding_fails_with_invalid_name(
+        looper, sdk_pool_handle, nodeSet, sdk_wallet_trustee, send_rs_encoding):
+    _, did = sdk_wallet_trustee
+    identifier, type, name, version = send_rs_encoding.split(':')
+    _id = identifier + ':' + type + ':' + "invalid_name" + ':' + version
+    request = build_get_rs_encoding_request(did, _id)
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_trustee, request)])
+    assert rep[0][1]['result']['seqNo'] is None
+
+
+def test_send_get_rs_encoding_fails_with_invalid_from(
+        looper, sdk_pool_handle, nodeSet, sdk_wallet_trustee, send_rs_encoding):
+    uuid_identifier = createUuidIdentifier()
+    _, did = sdk_wallet_trustee
+    identifier, type, name, version = send_rs_encoding.split(':')
+    _id = uuid_identifier + ':' + type + ':' + name + ':' + version
+    request = build_get_rs_encoding_request(did, _id)
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_trustee, request)])
+    assert rep[0][1]['result']['seqNo'] is None
+
+
+def test_send_get_rs_encoding_fails_with_invalid_version(
+        looper, sdk_pool_handle, nodeSet, sdk_wallet_trustee, send_rs_encoding):
+    _, did = sdk_wallet_trustee
+    identifier, type, name, version = send_rs_encoding.split(':')
+    _id = identifier + ':' + type + ':' + name + ':' + "2.0"
+    request = build_get_rs_encoding_request(did, _id)
+    rep = sdk_get_and_check_replies(looper, [sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_trustee, request)])
+    assert rep[0][1]['result']['seqNo'] is None
+
+
+def test_send_get_rs_encoding_fails_with_invalid_version_syntax(
```

Comment:
I think, that this and below tests can be parametrized

Reply:
I agree I will add a helper method to cut down on duplicate code.